kratos/tests/cpp_tests/input_output/test_logger.cpp

```diff
@@ -49,6 +49,7 @@ namespace Kratos {
 
         KRATOS_TEST_CASE_IN_SUITE(LoggerOutput, KratosCoreFastSuite)
         {
+            if(DataCommunicator::GetDefault().Rank() != 0) return;
```

Comment:
Maybe we could use these tests as well to check that nothing is being printed in the output of the remaining processes. Something in the line of:

```C++
KRATOS_TEST_CASE_IN_SUITE(LoggerOutput, KratosCoreFastSuite)
{
    std::stringstream buffer;
    LoggerOutput output(buffer);

    LoggerMessage message("label");
    message << "Test message with number " << 12 << 'e' << "00";
    output.WriteMessage(message);

    std::string expectedOutput = DataCommunicator::GetDefault().Rank() ? "" : "label: Test message with number 12e00";
    KRATOS_CHECK_C_STRING_EQUAL(buffer.str().c_str(), expectedOutput);
}
```